apps/nextjs/tests-e2e/tests/aila-chat/flexible-prompt.test.ts

```diff
@@ -0,0 +1,57 @@
+import { clerkSetup, setupClerkTestingToken } from "@clerk/testing/playwright";
```

Comment:
This is the original aila e2e test. I think the pattern for being able to test any prompt without fixtures is useful, but I don't directly have a use for it right now